logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/auth/LoginUseCase.kt

```diff
@@ -52,26 +50,22 @@ internal class LoginUseCaseImpl(
             validateUserHandleUseCase(cleanUserIdentifier) -> {
                 loginRepository.loginWithHandle(cleanUserIdentifier, password, shouldPersistClient, serverConfig)
             }
-            else -> return@suspending AuthenticationResult.Failure.InvalidUserIdentifier
+            else -> return@suspending LoginResult.Failure.InvalidUserIdentifier
         }.coFold({
             when (it) {
                 is NetworkFailure.ServerMiscommunication -> handleServerMiscommunication(it)
-                is NetworkFailure.NoNetworkConnection -> AuthenticationResult.Failure.Generic(it)
+                is NetworkFailure.NoNetworkConnection -> LoginResult.Failure.Generic(it)
             }
         }, {
-            if(shouldStoreSession) {
```

Comment:
This is now supposed to be handled by the new `AddAuthenticatedUser` but that's not wired up?

Reply:
Ah saw now that the PR is marked as WIP